src/model/schema.ts

```
@@ -1,13 +1,25 @@
-import { Node } from './node';
+import { Node as NoteNode, Element as NoteElement } from './types';
 
-type SchemaSpec = Record<string, NodeSpec>;
+/**
+ * `SchemaSpec` is a type that defines the structure of the schema.
+ */
+export type SchemaSpec = Record<string, NodeSpec>;
 type NodeSpec = ElementSpec | TextNodeSpec;
 type ElementSpec = { children: string };
 type TextNodeSpec = {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)